src/main/java/com/ams/restapi/timeConfig/TimeConfigNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.ams.restapi.timeConfig;
+
+class TimeConfigNotFoundException extends RuntimeException {
+    TimeConfigNotFoundException(Long id) {
```

Comment:
Not a big deal but consider changing id here to courseID to match with CourseInfoNotFoundException.java. They both are courseIDs.

Reply:
Changed, thanks Gabe.